### ISSUE TITLE
Diseases

### ISSUE THREAD

**Issue by TripleHeadedSheep** [TheCyberBrick/The-Betweenlands#471]
Had an idea about making 'hard' difficulty and 'hardcore' mode more challenging and interesting than just changing mob stats.
Players would be able to catch diseases from certain mobs (or plants, or other stuff perhaps). These diseases have a permanent effect until healed (maybe by sleeping, a potion, or some HL ingredients).
If a disease is left unattended for too long it will become more intense, buffing the effects a well perhaps adding new effects to it. Perhaps some diseases could lead to death when not cured for too long.

This idea might either be really interesting to work with, especially players who are looking for a proper challenge with a few different mechanics, or it might completely fail.
Feedback required.

**Comment by PetasiGit** [comment]
There should also be a way to keep diseases away, some sort of potion or vaccine to become immune after you've had each disease. Maybe you could catch a common cold from time to time as well, so that diseases won't be pointless after you get said vaccines. It could be part of herblore. 

I'd imagine you'd get diseases mostly from bugs or swamp hags, or by eating bad food, or maybe even just being in water. Could be interesting if you were more likely to catch a disease in certain biomes.

**Comment by TripleHeadedSheep** [comment]
Good idea to have some methods of preventing diseases. This offer potential for some interesting loot or even 'disease protected' world locations.
That way this gameplay element could also have ties to lower difficulties.

**Comment by TheCyberBrick** [comment]
While the idea sounds interesting, I don't think we should keep adding more and more mechanics to the player because this doesn't really add to "replayability" or diversity of the mod imo, but is more like something to micromanage like food, decay, corrosion and food sickness (already 4 things to constantly worry about!). I think we should focus more on progression instead, like the upgradeable armour for example. I could definitely see something like this as a "hard mode" addon or something like that at some point though

**Comment by TripleHeadedSheep** [comment]
Mhm, agreed. I put it down here for future reference. Many other things have priority.

**Comment by MokahTGS** [comment]
I have to agree that while this is an interesting idea, in our play testing, the BL already feels like a punishment enough.  In fact, I'm using that feeling as part of my mod pack.  Adding more punishment mechanics doesn't add anything at this point.

I'd rather see more added to research or alchemy systems.  Poke more critters in cages!!

**Comment by PetasiGit** [comment]
Once you're experienced with the BL, it's ridiculously easy. Hence why it would be an option for harder difficulty.
